tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Controllers/MatterController.cs

```diff
@@ -1138,29 +1115,7 @@ public IActionResult UpdateMetadata([FromBody]MatterMetdataVM matterMetdata)
                 #endregion   
 
                 try
-                {

```

Comment:
This code is not necessary as we are getting correct json request object from the client